hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -899,11 +898,13 @@ public String getMultipartKeyFSO(String volume, String bucket, String key, Strin
     final long volumeId = getVolumeId(volume);
     final long bucketId = getBucketId(volume,
             bucket);
-    long parentId =
-            OMFileRequest.getParentID(volumeId, bucketId, key, this);
-
-    String fileName = OzoneFSUtils.getFileName(key);
+    final String nonFSOMultipartKey =
+        getMultipartKey(volume, bucket, key, uploadId);
```

Comment:
Let's rename `nonFSOMultipartKey` to `multipartKey`. `nonFSOMultipartKey` name is quite confusing since we are dealing with FSO MPU key.

Reply:
TBH, I think `multipartKey` is ambitious. A `multipartKey` could be a FSO multipartKey or nonFSO multipartKey, since here we are dealing with FSO, that's why I intentionally name it as `nonFSOMultipartKey` so that the reader can understand we should not directly return this value.

If you think `multipartKey` is conventionally mean nonFSO multipartKey, I can rename it. Pls let me know. Thanks

Reply:
Let's make the following convention:
- `multipartKey` is the key in `multipartInfoTable` (regardless whether the underlying bucket is FSO or OBS) that's created in `initiateMultiaprtUpload`
- `multipartOpenKey` is the key in `openKeyTable` / `openFileTable` that's created in `initiateMultipartUpload`
- `multipartOpenPartKey` is the key in `openKeyTable` / `openFileTable` that's created in `OMKeyCreateRequest` when uploading a part key.

You can refer to https://issues.apache.org/jira/browse/HDDS-9098. So I suggest to rename it to `multipartKey`.

Reply:
I think we are talking about different problems. Here is about the multipartKey for FSO or non-FSO. The non-FSO multipartKey is of this format: `volume/bucket/key/uploadId`, while the FSO one is: `volumeId/bucketId/parentId/fileName/uploadId`.

But anyway, I will just change it to `multipartKey` for simplicity.